README.md

```diff
@@ -1 +1,236 @@
-# Todo-list
\ No newline at end of file
+# Awesome-books
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [To Do list: list structure] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+In this project, I will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server.
+
+**[Awesome Books]** is a basic website that allows users to add/remove books from a list
+
```

Comment:
- [ ] [OPTIONAL] You used the correct `README` template, 👍🏼 but I'd suggest you take a closer look at it, as this is not an `Awesome Books` project 😃